framework/aster-frame/src/task/task.rs

```diff
@@ -273,32 +277,32 @@ impl TaskOptions {
             current_task.func.call(());
             current_task.exit();
         }
-        let result = Task {
+
+        let mut result = Task {
```

Comment:
The original variable name is poorly chosen. Replace `result` with `task` or `new_task` instead.